cmd/rawkv-linearizability/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/pingcap/tipocket/cmd/util"
+	"github.com/pingcap/tipocket/pkg/check/porcupine"
+	"github.com/pingcap/tipocket/pkg/cluster"
+	"github.com/pingcap/tipocket/pkg/control"
+	"github.com/pingcap/tipocket/pkg/core"
+	test_infra "github.com/pingcap/tipocket/pkg/test-infra"
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
+	"github.com/pingcap/tipocket/pkg/verify"
+	rawkvlinearizability "github.com/pingcap/tipocket/tests/rawkv-linearizability"
+)
+
+var (
+	keyStart        = flag.Int("KeyStart", 0, "the start of the key")
+	keyNum          = flag.Int("KeyNum", 100000, "the key range number")
+	readProbability = flag.Int("ReadProbability", 60, "the probaility of read request")
+	writeProbaility = flag.Int("WriteProbaility", 35, "the probaility of write request, the remaining part is the probaility of delete request")
+	valueNum10KB    = flag.Int("ValueNum10KB", 400, "10KB value kind number")
+	valueNum100KB   = flag.Int("ValueNum100KB", 400, "100KB value kind number")
+	valueNum1MB     = flag.Int("ValueNum1MB", 200, "1MB value kind number")
+	valueNum5MB     = flag.Int("ValueNum5MB", 40, "5MB value kind number")
+)
+
+func main() {
+	flag.Parse()
+
+	var checkers []core.Checker
+	checkers = append(checkers, porcupine.Checker{})
+	// TODO should add more checker
```

Comment:
Is here enough now?

Reply:
No...I will add one more checker at least